lib/ClangImporter/ImportDecl.cpp

```diff
@@ -2522,6 +2522,15 @@ namespace {
           VarDecl *pointeeProperty =
               synthesizer.makeDereferencedPointeeProperty(
                   getterAndSetter.first, getterAndSetter.second);
+
+          // Import the attributes from clang decl of dereference operator to
+          // synthesized pointee property.
+          FuncDecl *getterOrSetterImpl = getterAndSetter.first
```

Comment:
Are there cases where we only have a setter but no getter?

Reply:
I don't think so. For operator* we must have a getter, setter is optional. Is that correct @egorzhdan ?
I just saw similar logic inside `makeDereferencedPointeePropert` function.

Reply:
We might have a "setter" and no "getter" (might not be the best terminology – I tried to use similar terms to Obj-C interop in this logic but that might be a bit confusing) if there is only a non-const overload of `operator*`:
```cpp
struct Iterator {
private:
  int value = 123;
public:
  int &operator*() { return value; }
};
```